src/cmd-kola

```diff
@@ -36,10 +36,11 @@ blacklist_path = "src/config/kola-blacklist.yaml"
 if os.path.isfile(blacklist_path):
     with open(blacklist_path) as f:
         blacklist = yaml.safe_load(f)
-        for obj in blacklist:
-            print(f"⚠️  Skipping kola test pattern \"{obj['pattern']}\":")
-            print(f"⚠️  {obj['tracker']}")
-            blacklist_args += ['--blacklist-test', obj['pattern']]
+        if blacklist is not None:
+            for obj in blacklist:
```

Comment:
```suggestion
            for obj in blacklist or []:
```

Reply:
That would render `if blacklist is not None` useless

Reply:
Thanks, updated! :arrow_down: I wrapped it in parentheses to make it more obvious what's going on there.